src/components/OwnerHome/ownerHomeService.js

```diff
@@ -54,6 +54,33 @@ export default class OwnerHomeService {
     return response;
   };
 
+  updateRealizationTime = async (tokenNumber, storeId) => {
+    let startDate = new Date();
+    startDate.setHours(0);
+    startDate.setMinutes(0);
+    startDate.setSeconds(0);
+    startDate.setMilliseconds(0);
+
+    let currentTime = new Date();
+
+    let filter = {
+      where: {
+        and: [
+          { store_id: storeId },
+          { token_number: tokenNumber },
+          { date: { gte: startDate } }
+        ]
+      }
+    };
+
+    const tokens = await axios.get(`${API_URL}/tokenBookings?filter=${JSON.stringify(filter)}`);
+
+    await Promise.all(tokens.data.map(async (eachToken) => {
```

Comment:
since you're not interested in the array of resolved promises, you can use Promise.each here - http://bluebirdjs.com/docs/api/promise.each.html

Reply:
Actually, wanted to execute the api calls in parallel (if at all it returns more than one)